docs-src/content/getting-started/quickstart.md

```diff
@@ -5,7 +5,7 @@ weight: 2
 
 To evaluate Cobalt's speech recognition, you can access the [Cubic web demo](https://demo-cubic.cobaltspeech.com:8080) or can download a command-line interface built using our Go SDK to specify an audio file or list of files and get transcripts back from a running instance of cubicsvr. The [source code for cubic-cli](https://github.com/cobaltspeech/cubic-cli) is available to use as an example client.
 
-The command-line tool can call a local instance of Cubic (see [Installing the cubicsvr Image](cubic_docker.md)) or the demo server.
+The command-line tool can call a local instance of Cubic (see [Installing the Cubicsvr Image](/sdk-cubic/getting-started/cubic_docker)) or the demo server.
 
 ```bash
 # Local instance
```

Comment:
missing the "./" at the beginning of the call

Reply:
fixed.  Of course, these two commands are equivalent, but it's good to be consistent so as not to cause confusion!